### ISSUE TITLE
`SIM118` should detect `not in` as well

### ISSUE THREAD

**Issue by dosisod** [astral-sh/ruff#5989]
Last issue for now!

Ruff only detects `in`, but it should detect `not in` as well:

```python
d = {}

if "key" in d.keys():
    pass

if "key" not in d.keys():
    pass
```

Ruff output:

```
$ ruff x.py
x.py:3:4: SIM118 [*] Use `"key" in d` instead of `"key" in d.keys()`
```

Expected:

```
x.py:3:4: SIM118 [*] Use `"key" in d` instead of `"key" in d.keys()`
x.py:6:4: SIM118 [*] Use `"key" not in d` instead of `"key" not in d.keys()`
```
